README.md

```diff
@@ -30,7 +45,7 @@ yarn add --dev eslint@^8.10.0 \
                eslint-plugin-node@^11.1.0
 ```
 
-2. Add `"extends": "strapi"` to your `.eslintrc`
+2. Add `"extends": "strapi/back"` to your `.eslintrc`
```

Comment:
@Elhebert What do you think about naming this "node" instead? Instead of `back` and `front` I'd think that "node" and "browser" are more appropriate because it refers to a specific set of features supported.